crates/wash-lib/README.md

```diff
@@ -1,7 +1,8 @@
 [![crates.io](https://img.shields.io/crates/v/wash-lib.svg)](https://crates.io/crates/wash-lib)&nbsp;
-# wash Libary
+
+# wash Library
 
 A crate that implements the functionality behind the wasmCloud shell
 
-The `wash` command line interface [wash](https://github.com/wasmcloud/wash) is a great place
-to find examples on how to fully utilize this library.
\ No newline at end of file
+The `wash` command line interface [wash](https://github.com/wasmCloud/wasmCloud/tree/main/crates/wash-cli) is a great place
+to find examples on how to fully utilize this library.
```

Comment:
```suggestion
to find examples on how to fully utilize this libary.
```

Reply:
Alternatively this would keep us consistent

Reply:
That "commit suggestion" button is looking awfully tempting